specs/container/readme.md

```diff
@@ -38,4 +38,59 @@ type ContainerModule interface {
     // Ps displays the processes running inside a container
     Ps(ID string) (error)
 }
-```
\ No newline at end of file
+```
+
+## OCI bundle
+
+The oci bundle must define `rootfs` and a `config.json`. currently flist defines only the `rootfs` of a container. There are no attached meta to define entrypoint, env variables, exposed ports, etc ...
+
+Since the flist is a tar that container only one file (the rootfs db), it's okay we add more files to the tar. I suggest we add custom meta file that contains
+the missing pieces of the bundle that we need to construct the final `config.json` file.
+
+### Flist meta
+
+Part of the `config.json` entries that can be defined byt the user during the flist build stage or when creating a container:
+
+- process.terminal
+- process.user
+- process.args
+- process.env
+- process.cwd
+- hostname
```

Comment:
Hostname should be defined in runtime, hence it should be moved to the other section

Reply:
I'm not sure about that. Why exactly this must be chosen by the runtime ?
I think use should be able to choose the hostname of their container and use the hostname to address other containers.